src/auto/network_proxy_settings.rs

```diff
@@ -1,27 +1,27 @@
 // This file was generated by gir (https://github.com/gtk-rs/gir)
-// from gir-files (https://github.com/gtk-rs/gir-files)
+// from gir-files (https://github.com/gtk-rs/gir-files.git)
 // DO NOT EDIT
 
 #[cfg(any(feature = "v2_16", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v2_16")))]
 use glib::translate::*;
-use webkit2_sys;
 
-glib_wrapper! {
+glib::wrapper! {
     #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-    pub struct NetworkProxySettings(Boxed<webkit2_sys::WebKitNetworkProxySettings>);
+    pub struct NetworkProxySettings(Boxed<ffi::WebKitNetworkProxySettings>);
 
     match fn {
-        copy => |ptr| webkit2_sys::webkit_network_proxy_settings_copy(mut_override(ptr)),
-        free => |ptr| webkit2_sys::webkit_network_proxy_settings_free(ptr),
-        get_type => || webkit2_sys::webkit_network_proxy_settings_get_type(),
+        copy => |ptr| ffi::webkit_network_proxy_settings_copy(mut_override(ptr)),
+        free => |ptr| ffi::webkit_network_proxy_settings_free(ptr),
+        type_ => || ffi::webkit_network_proxy_settings_get_type(),
     }
 }
 
 impl NetworkProxySettings {
-    #[cfg(any(feature = "v2_16", feature = "dox"))]
+    #[doc(alias = "webkit_network_proxy_settings_add_proxy_for_scheme")]
```

Comment:
It looks like it's missing the constructor `webkit_network_proxy_settings_new`.
Maybe it's not in the `gir` file?

Reply:
I'm not familiar with a lot of details of gir-files, but looks to me like the constructor is present: https://github.com/gtk-rs/gir-files/blob/master/WebKit2-4.0.gir#L6745

Reply:
That's because it is also [ignored](https://github.com/gtk-rs/webkit2gtk-rs/blob/c70c9fb34fc0e536a90dcf99293c9f7f57469e0e/Gir.toml#L217).

Reply:
Ah, makes sense. Fixed